client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -86,6 +65,28 @@ const EmailProvidersStackedComparison = ( {
 		};
 	} );
 
+	const canPurchaseGoogleWorkspace = useSelector( canUserPurchaseGSuite );
```

Comment:
This is me splitting hairs but I would still refer to G Suite here:

```suggestion
	const canPurchaseGSuite = useSelector( canUserPurchaseGSuite );
```

That way the name is consistent with the variable, and we avoid mixing names like this which could be confusing for developers not familiar with this part of our code:

```javascript
const isGSuiteSupported = canPurchaseGoogleWorkspace && resolvedDomainName && hasGSuiteSupportedDomain( [ domain ] );
```

Reply:
😄 All fine, I'm renaming it